src/debug.cpp

```diff
@@ -704,8 +704,8 @@ struct OutputDebugStreamA : public std::ostream {
 #endif
 
 struct DebugFile {
-    ~DebugFile();
```

Comment:
We still want to have a destructor that calls `deinit()` in it, right? In case `deinitDebug()` isn't called for whatever reason..